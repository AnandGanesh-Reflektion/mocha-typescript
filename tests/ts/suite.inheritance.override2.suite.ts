import { suite, test } from "../../index";
import { assert } from "chai";

@suite class PartialTest {

  @test.skip "test must be implemented by derived suites"() {
  }
}

@suite class CompleteTest extends PartialTest {

  @test "test must be implemented by derived suites"() {

    assert.ok(true);
  }
}
