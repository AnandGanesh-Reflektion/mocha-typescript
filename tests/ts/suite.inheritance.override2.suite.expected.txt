
  PartialTest
    - test must be implemented by derived suites

  CompleteTest
    âœ“ test must be implemented by derived suites

  1 passing
  1 pending
